test/runtests.jl

```diff
@@ -21,13 +21,27 @@ end
     end
 end
 
-@testset "variable types test" begin
+@testset "miscellaneous tests" begin
```

Comment:
This normally wouldn't pass JuMP-style code review, but I'll let it go since MPB is going away soon. Tests should ideally cover one thing at a time.